crowbar_framework/app/views/barclamp/dns/_edit_attributes.html.haml

```diff
@@ -3,7 +3,12 @@
     = header show_raw_deployment?, true
 
   .panel-body
-    = string_field :domain
+
+    %fieldset
+      = string_field :domain, :disabled => "disabled"
```

Comment:
wouldnt this make the domain field always disabled, even on the first dns barclamp proposal creation?

Could we play with the `@proposal.active?` var to disable it once it has been deployed?

```
      = string_field :domain, :disabled => @proposal.active?
```